src/components/LoginPage/LoginPage.jsx

```diff
@@ -5,26 +5,89 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import { Form, Col, Button } from 'react-bootstrap';
 import { useHistory } from 'react-router-dom';
 
-const StyledTitle = styled.h2`
-   margin: 20px 0px 15px 20px;
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEyeSlash } from '@fortawesome/free-solid-svg-icons';
+
+import Logo from './Logo.svg';
+
+const StyledAll = styled.div`
+   display: flex;
+   flex-direction: column;
+   justify-content: space-between;
 `;
 
+const StyledTitle = styled.h2`
+   display: flex;
+   justify-content: center;
+   color: #024e6b;
+   font-size: 48px;
+   font-weight: 700;
+`;
+const StyledImage = styled.img`
+   display: flex;
+   justify-content: center;
+`;
+const StyledText = styled.div`
+   margin-left: 33px;
+   margin-bottom: 3px;
+   font-size: 18px;
+   font-weight: 700;
+   color: #024e6b;
+`;
```

Comment:
Make sure this matches the mockups. The text for email and password looks different on the mockups.